engine/containerd/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: containerd integration with Docker Engine
+description: Viewing the containerd integration in Docker for image and file system management
+keywords: Docker, containerd, engine, image store, lazy-pull
+toc_min: 1
+toc_max: 2  
+---
+
+This page presents information about the on-going integration of containerd for image and file system management in the Docker Engine.
+
+> **Warning**
+>
+> The containerd image store management feature is currently experimental. This may change or be removed from future releases.
+{: .warning }
+
+## 2022-09-01
+Beta release support of containerd for image management as an Experimental Feature in Docker Desktop 4.12.0.
+
+### New
+Initial implementation of the Docker commands: `run`, `commit`, `build`, `push`, `load`, `search` and `save`.
+
+### Known issues
+- The containerd integration requires Buildx version 0.9.0 or newer.  On Docker Desktop for Linux (DD4L), validate the locally installed version meets this requirement.  If an older version is installed, the Docker daemon will report an error: `multiple platforms feature is currently not supported for docker driver. Please switch to a different driver`.
```

Comment:
(FWIW: a buildx v0.9.1 will be included in the next 20.10.18 patch release);

Perhaps for now, we should also link to how to manually install a newer version of buildx; https://docs.docker.com/build/buildx/install/#manual-download

Reply:
Thanks, just added the link to manually update Buildx.